src/assets/css/librarybrowser.css

```diff
@@ -584,6 +653,9 @@
     white-space: nowrap;
     text-overflow: ellipsis;
     text-align: left;
+    min-width: 0;
+    max-width: 100%;
+    overflow: hidden;
```

Comment:
I hate using this file as a catch-all for page styles, but we can't really change that yet.